pyo3-macros-backend/src/pyclass.rs

```diff
@@ -1916,7 +1916,7 @@ fn pyclass_richcmp_simple_enum(
     let eq = options.eq.map(|eq| {
         quote_spanned! { eq.span() =>
             let self_val = self;
-            if let ::std::result::Result::Ok(other) = #pyo3_path::types::PyAnyMethods::downcast::<Self>(other) {
+            if let ::std::result::Result::Ok(other) = other.cast::<Self>() {
```

Comment:
it's nice to see the method avoiding the need for some painful call syntax!

Reply:
Yes! This was always bugging me.